src/modules/taskManager/taskForm/TaskFormView.js

```diff
@@ -28,13 +28,19 @@ const TaskFormView = React.createClass({
   },
 
   submitTask() {
-    let formData = this.refs.taskForm.values;
-    console.log(formData);
-    this.props.dispatch(TaskFormState.post(formData));
-    // this.props.dispatch(NavigationState.pushRoute({
-    //   key: 'SubmitTask',
-    //   title: 'Submit Task'
-    // }));
+    if (this.refs.taskForm.values.taskName && 
```

Comment:
I think it will be more readable if this conditional was in it's own variable like `formValid`. then you could do `if (!formValid) {return;}` and drop the `else`